### PR TITLE
Add firebase configs + Select country screen

### PR DESCRIPTION
**Pull request by manhduc-nguyen**
[https://github.com/notional-labs/beanstalk-mobile/issues/10](https://github.com/notional-labs/beanstalk-mobile/issues/10)
Screen Shot 2022-01-21 at 22 11 10
Screen Shot 2022-01-21 at 22 11 17

### CODE REVIEW

File: lib/app/cubit/choose_country/choose_country_cubit.dart [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+class ChooseCountryCubit extends Cubit<ChooseCountryState> {
+  ChooseCountryCubit() : super(ChooseCountryInitialState());
+
+  void fetchCountries() async {
+    emit(ChooseCountryLoadingState(state.model));
+
+    // fast implement for now
+    await RemoteConfig.instance.fetchAndActivate();
+    final countries =
+        RemoteConfig.instance.getString('available_countries').split(',');
```
Comment (mduccc) [inline lib/app/cubit/choose_country/choose_country_cubit.dart:18]: Should be use `Array` instead `String` on Firebase in the future

File: lib/app/page/choose_country/choose_country_page.dart [hunk 2]
```diff
@@ -7,9 +16,127 @@ class ChooseCountryPage extends StatefulWidget {
...
+              )),
+              Positioned.directional(
+                  top: 66,
+                  start: 0,
+                  end: 0,
+                  bottom: 0,
+                  textDirection: TextDirection.ltr,
+                  child: Column(
+                    children: [
+                      Text(
```
Comment (mduccc) [inline lib/app/page/choose_country/choose_country_page.dart:70]: Consider use`DSPrimaryAppBar`

**Review by mduccc — APPROVED** [review]

File: lib/app/page/choose_country/choose_country_page.dart (hunk 2 above)
Comment (manhduc-nguyen) [reply lib/app/page/choose_country/choose_country_page.dart]: great

File: lib/app/cubit/choose_country/choose_country_cubit.dart (hunk 1 above)
Comment (manhduc-nguyen) [reply lib/app/cubit/choose_country/choose_country_cubit.dart:18]: yes, no need complicated implementation for now

**Pull request merged**